.eslintrc.js

```diff
@@ -6,40 +24,145 @@ module.exports = {
     env: {
         jest: true,
     },
-    settings: {
-        'import/resolver': {
-            node: {
-                extensions: ['.js', '.website.js', '.desktop.js', '.native.js', '.ios.js', '.android.js', '.config.js'],
-            },
-        },
-    },
     globals: {
         __DEV__: 'readonly',
     },
-    rules: {
-        'no-restricted-imports': [
-            'error',
-            {
-                paths: [
-                    {
-                        name: 'react-native',
-                        importNames: ['useWindowDimensions', 'StatusBar', 'TouchableOpacity', 'TouchableWithoutFeedback', 'TouchableNativeFeedback', 'TouchableHighlight', 'Pressable'],
-                        message: [
-                            '',
-                            "For 'useWindowDimensions', please use 'src/hooks/useWindowDimensions' instead.",
-                            "For 'TouchableOpacity', 'TouchableWithoutFeedback', 'TouchableNativeFeedback', 'TouchableHighlight', 'Pressable', please use 'PressableWithFeedback' and/or 'PressableWithoutFeedback' from 'src/components/Pressable' instead.",
-                            "For 'StatusBar', please use 'src/libs/StatusBar' instead.",
-                        ].join('\n'),
+    overrides: [
+        {
+            files: ['*.js', '*.jsx', '*.ts', '*.tsx'],
+            rules: {
+                'react-native-a11y/has-accessibility-hint': ['off'],
+                'react-native-a11y/has-valid-accessibility-descriptors': [
+                    'error',
+                    {
+                        touchables: ['PressableWithoutFeedback', 'PressableWithFeedback'],
```

Comment:
[The doc for the rule](https://github.com/FormidableLabs/eslint-plugin-react-native-a11y/blob/master/docs/rules/has-valid-accessibility-descriptors.md) says that this rule doesn't take any arguments...

does specifying `touchables` work here?

Reply:
That's a good question! I asked Michał from SWM who added it to `.eslintrc.js` file.

Reply:
Without `touchables` option it doesn't work, the weird part is that it isn't documented, but I found the code responsible for that.

[has-valid-accessibility-descriptors rule implementation](https://github.com/FormidableLabs/eslint-plugin-react-native-a11y/blob/master/src/rules/has-valid-accessibility-descriptors.js#L34) calls [isTouchable function](https://github.com/FormidableLabs/eslint-plugin-react-native-a11y/blob/master/src/util/isTouchable.js#L21) which actually uses the `touchables` option 😄